app/views/declaration/summary/transport_containers_section.scala.html

```diff
@@ -19,12 +19,12 @@
 
 @(containers: Seq[Container])(implicit messages: Messages)
 
-@components.summary_list(Some(messages("declaration.transportInfo.containers.title"))) {
+@components.summary_list(Some(messages("declaration.transportInformation.containers.title"))) {
```

Comment:
@PatrykRudnicki here could be conflict with your ticket

Reply:
I removed this section, so it's not a big deal